pyflakes/test/test_type_annotations.py

```diff
@@ -121,6 +121,23 @@ def f(self, x):  # type: (str) -> str
             def f(self, x): return x
         """)
 
+    def test_aliased_import(self):
```

Comment:
looks like you'll need to guard this test to be python 3 only because it uses `async def`

Reply:
@asottile thanks, I didn't even notice that. fixed